photo-app/app/models/payment.rb

```diff
@@ -0,0 +1,19 @@
+class Payment < ApplicationRecord
+  attr_accessor :card_number, :card_cvv, :card_expires_month, :card_expires_year
+  belongs_to :user
```

Comment:
Please organize your code properly. Have an extra space for each section. don't just jumble up together. That makes it hard to read